plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -1834,4 +1834,17 @@ public String toString() {
         vmBuilder.append("</domain>\n");
         return vmBuilder.toString();
     }
+
+    public static class GenericXML {
```

Comment:
In case the previous comment works, this class won't be needed

Reply:
Removing redundant class